x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/logic/import/create_rules_stream_from_ndjson.ts

```diff
@@ -103,6 +104,25 @@ export const sortImports = (): Transform => {
   );
 };
 
+export const migrateRules = (): Transform => {
```

Comment:
Want to support use case of users exporting their rules in 8.10 and importing in 8.11+ without them needing to manually update this field.